analysis/wwz_etaphi_plotting/full_run3_run.sh

```diff
@@ -0,0 +1,7 @@
+# Some example run commands for R3 configs
+# ALL RUN SCRIPTS ARE TO BE RUN LOCALLY AT UAF!
+
+#################################################################################################################################
+
+# 2022 + 2023
+time python run_wwz4l.py ../../input_samples/cfgs/wwz_analysis/samples_4lskim_2022_v1.cfg,../../input_samples/cfgs/wwz_analysis/samples_4lskim_2023_v1.cfg -o r3_wwz_histos_noSyst -x futures -n 100
```

Comment:
I'd probably just stick this into the R2 script and just have one script since it's just one line each (just to keep this dir lighter weight)